.github/workflows/node.yml

```diff
@@ -269,3 +271,45 @@ jobs:
                       node/test-report*.html
                       utils/clusters/**
                       benchmarks/results/**
+
+    test-modules:
+        if: github.event.pull_request.head.repo.owner.login == 'valkey-io'
+        # environment: AWS_ACTIONS
+        name: Running Module Tests
+        runs-on: [self-hosted, linux, ARM64]
+        timeout-minutes: 15
+
+        steps:
+            - name: Setup self-hosted runner access
+              run: sudo chown -R $USER:$USER /home/ubuntu/actions-runner/_work/valkey-glide
+
+            - uses: actions/checkout@v4
+              with:
+                  submodules: recursive
+
+            - name: Use Node.js 16.x
+              uses: actions/setup-node@v3
+              with:
+                  node-version: 16.x
+
+            - name: Build Node wrapper
+              uses: ./.github/workflows/build-node-wrapper
+              with:
+                  os: ubuntu
+                  named_os: linux
+                  arch: arm64
+                  target: aarch64-unknown-linux-gnu
+                  github-token: ${{ secrets.GITHUB_TOKEN }}
+
+            - name: test
+              run: npm run test-modules --tls -- --cluster-endpoints=${{ secrets.MEMDB_MODULES_ENDPOINT }}
```

Comment:
Shouldn't be
```suggestion
              run: npm run test-modules -- --tls --cluster-endpoints=${{ secrets.MEMDB_MODULES_ENDPOINT }}
```

Reply:
i've tried that and adding it after the -- does not get recognized and causes an error

Reply:
it is weird that these parameters are parsed in different ways

Reply:
I guess TLS isn't parsed at all and ignored (not used)

Reply:
that seems to be the case, getting connection errors in the tests, including `useTLS` as a client config override also failed